specifyweb/frontend/js_src/lib/components/FormEditor/Create.tsx

```diff
@@ -245,6 +246,10 @@ function ChooseName({
   );
 
   const navigate = useNavigate();
+
+  const { appResource } = React.useContext(FormEditorContext)!;
+  const newName = appResource.get('name');
```

Comment:
rename this variable to `appResourceName` or something like that - so that it's not confused with view name